### ISSUE TITLE
Obsidian footnotes are not working

### ISSUE THREAD

**Issue by src4026** [alangrainger/share-note#53]
Hi, I recently started trying out this plugin. In doing so, I seem to have come across an issue which could be a bug.
I have a couple of footnotes in my document using the default obsidian syntax for footnotes, but whenever I click on the footnote link, it takes me to another page stating "Unable to decrypt using this key". There should be a way to link to it internally on the page itself, rather than whatever the service is misinterpreting it as.

I hope this gets resolved soon, cheers!

**Comment by ksdavidc** [comment]
Not working for me either. if I am encrypting. Unencrypted, it works okay.

**Comment by Mara-Li** [comment]
Even with no encryption, I don't have the footnote rendered :/

**Comment by alangrainger** [comment]
These are now working - sorry for the delay in fixing this!

Demo:

https://share.note.sx/nsbv5u5t#Ulh3gL7HBJnl/7sp7FyuTPPp+5iBXvm/cjWsS5HIe1k

**Comment by ksdavidc** [comment]
The footnotes are appearing, but links to the reference links do not seem to be clickable. The link back to the reference works.

**Comment by alangrainger** [comment]
@ksdavidc  I provided a demo showing the links to the footnote working. 

Can you provide the markdown which is failing to produce working links for you?

**Comment by ksdavidc** [comment]
sorry to necropost. This is FYI, in case anyone comes across this:

For me, it appears the problem was that there was a space before the footnote symbol. Strangely, once I fixed it, it then worked even if I reverted!? It was the only change I made...


1. Does not work on page, but works in obsidian. [^1]
2. works in  both.[^2]

[^1]: note the space. The particular instance was also a numbered list.
[^2]: no space
